db/migrate/20221116104013_add_capabilities_to_physical_storage_models.rb

```diff
@@ -0,0 +1,8 @@
+class AddCapabilitiesToPhysicalStorageModels < ActiveRecord::Migration[6.1]
+  def change
+    add_column :physical_storage_families, :abstract_capability_values, :jsonb, :default => {}
+    add_column :physical_storages, :enabled_capability_values, :jsonb, :default => {}
+    add_column :storage_resources, :enabled_capability_values, :jsonb, :default => {}
+    add_column :storage_services, :enabled_capability_values, :jsonb, :default => {}
```

Comment:
I think unless we are planning on storing other types of capabilities here (e.g. are we also going to have a `concrete_capability_values` or a `disabled_capability_values` column) we should keep this simple and call it `capabilities` in line with the other models

Reply:
Field names have changed to capabilities.